docs/wsrep-system-index.md

```diff
@@ -1240,6 +1240,42 @@ As of *Percona XtraDB Cluster* 8.0.26-16, this variable is deprecated and may be
 Defines whether unique key checking is done for applier threads.
 This is disabled by default.
 
+### `wsrep_SR_store`
+
+| Option         | Description        |
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep-sr-store`` |
+| Config File:   | Yes                |
+| Scope:         | Global            |
+| Dynamic:       | No                 |
+| Default Value: | ``table`` |
+
+Defines storage for streaming replication fragments. The default value is `table`. To disable this variable, specify `none`.
```

Comment:
The available values are `table`, the default value, and `none`, which disables the variable.

Reply:
updated